pulp_rpm/app/constants.py

```diff
@@ -127,7 +127,7 @@
     ARCH='arch'
 )
 
-CR_UPDATE_REFERENCE_ATTRS = SimpleNamespace(
+PULP_UPDATE_REFERENCE_ATTRS = SimpleNamespace(
```

Comment:
This change is unrelated to the issue for which PR is created.
Could you submit it separately. It likely solves this issue, btw, https://pulp.plan.io/issues/5285.

Reply:
I did this change because some tests were breaking, but I agree it is not related to this PR.

Reply:
the new PR is here:
https://github.com/pulp/pulp_rpm/pull/1425